CHANGELOG.md

```diff
@@ -250,6 +250,8 @@ The format is based on [Keep a Changelog].
 * Annotations or usage or `selectrum-candidate-display-suffix`
   property in file completions were overwritten for directories and
   not displayed, which has been fixed ([#256], [#255]).
+* Assigned `this-comamnd` to value of `selectrum--last-command` in
+  `selectrum-repeat` ([#438]).
```

Comment:
In the changelog we try to explain what has changed for the user not the actual changes. For example: `selectrum-repeat` did not set `this-command` when calling the last command which has been fixed.

Reply:
Oooh, my bad. Fixed it now :-).